src/api/route-services/projects.js

```diff
@@ -65,6 +66,45 @@ class ProjectsService {
     }
   }
 
+  async getProjects() {
+    const params = {
+      TableName: experimentService.experimentsTableName,
+      AttributesToGet: ['projectId'],
+    };
+
+    const dynamodb = createDynamoDbInstance();
+    const response = await dynamodb.scan(params).promise();
+    const projectIds = [];
+    response.Items.forEach((entry) => {
+      if (entry.projectId) {
+        const newEntry = convertToJsObject(entry);
+        projectIds.push(newEntry.projectId);
+      }
+    });
+    const projects = [];
+
+    if (projectIds.length) {
+      return new Promise((resolve) => {
+        projectIds.forEach(async (id) => {
+          const newData = await this.getProject(id);
```

Comment:
What do you think about using `batchGetItem` instead? To avoid the overhead of each individual http request

Reply:
I would still need to call batchgetitem for each projectId, you think that will make a difference? I thought it would be good to reuse the getProject function that we already have.

Reply:
With batchGetItem you should be able to pass an array with all the projectIds and receive an array with all the responses, so it would be only one time.

Reply:
it could be incorporated as an internal getProjectsFromIds or something like that perhaps.